### PR TITLE
update README.md

### PR DESCRIPTION
**Pull request by ctrombley**
Add some additional info to the README.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,55 @@
...
+		APIKey: "<NEWRELIC_API_KEY>",
+	}
+
+    client := newrelic.New(config)
+
+	config := newrelic.Config{
+		APIKey: "<NEWRELIC_API_KEY>",
+	}
+
+	apps, err := client.ListApplications()
```
Comment (sanderblue) [inline README.md]: Should we update the indention here to make easier to copy and paste? 😎

File: README.md [hunk 2]
```diff
@@ -1,3 +1,55 @@
...
+The GoDoc link below contains the full list of available methods.
+
+## Contributing
+Contributions are what make the open source community such an amazing place to be learn, inspire, and create. Any contributions you make are greatly appreciated.
+
+1. Fork the Project
+1. Create your Feature Branch (git checkout -b feature/AmazingFeature)
+1. Commit your Changes (git commit -m 'Add some AmazingFeature')
+1. Push to the Branch (git push origin feature/AmazingFeature)
+1. Open a Pull Request
```
Comment (sanderblue) [inline README.md]: Even though markdown orders this correctly, this kind of confuses me as a reviewer. Not necessarily a blocker, but would it better to use the correct numbers in the markdown or format the numbered list in a different way?

File: README.md [hunk 3]
```diff
@@ -1,3 +1,55 @@
...
+
+### Running tests
+```bash
+go test -v ./...
+```
+
+## License
+[MIT](https://choosealicense.com/licenses/mit/)
```
Comment (jthurman42) [inline README.md]: License file in this repo is Apache 2.0, not MIT.  Should link to the file in the repo.

**Review by jthurman42 — CHANGES_REQUESTED** [review]

**Review by sanderblue — CHANGES_REQUESTED** [review]

**Review by sanderblue — DISMISSED** [review]

File: README.md [hunk 4]
```diff
@@ -1,3 +1,63 @@
...
+4. Push to the Branch (git push origin feature/AmazingFeature)
+5. Open a Pull Request
+
+### Running tests
+```bash
+go test -v ./...
+```
+
+## License
+[Apache-2.0](https://github.com/paultyng/go-newrelic/blob/master/LICENSE)
```
Comment (jthurman42) [inline README.md]: This should be a local ref, not the HTTP path

**Review by jthurman42 — DISMISSED** [review]

**Review by jthurman42 — APPROVED** [review]

**Pull request merged**